Burn subtitles

howardbaik opened the pull request:
<!--This PR Template was modified from https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/.github/PULL_REQUEST_TEMPLATE.md-->

#### Purpose/implementation Section

I wanted to add subtitles to Loqui so I experimented with `ari_burn_subtitles()`and fixed a bug. 

#### What changes are being implemented in this Pull Request?

The `system()` command specified the wrong path to the video output. It was specifying the same path for both the input and the output, which ffmpeg didn't like. I changed the path to the video output so we can have different paths for the input and output.

Commit:
`ari_burn_subtitles()`: Fix destination of output in system command

Commit:
Document `ari_subtitles()` and `ari_burn_subtitles()`

Commit:
Fix dependency issue

howardbaek commented:
Not sure what the conflict is. Github is showing me a conflict in the NAMESPACE https://github.com/jhudsl/ari/pull/52/conflicts

howardbaik committed:
Fix merge conflicts when git pull-ing from ariExtra-immigration branch

Merge branch 'ariExtra-immigration' of https://github.com/jhudsl/ari into burn-subtitles

# Conflicts:
#	NAMESPACE

seankross commented:
Hey Howard, is there any reason we shouldn't be comparing this PR against the main branch?

howardbaik commented:
@seankross Technically, this can be compared against the main branch. I'm comparing it against the `ariExtra-immigration` branch because [Loqui](https://loqui.fredhutch.org/) uses ari functions in that branch and not the main branch.

seankross commented on R/ari_burn_subtitles.R:
```diff
@@ -4,20 +4,24 @@
-ari_burn_subtitles <- function(video, srt, verbose = FALSE) {
+#' @export
+ari_burn_subtitles <- function(input_video, srt,
+                               output_video = tempfile(fileext = ".mp4"),
```
output_video should not have a default value

howardbaik committed:
Merge branch 'ariExtra-immigration' into burn-subtitles

howardbaik committed:
Get rid of default argument for `output_video`

howardbaik committed:
Documentation stuff
